application/app/services/download/download_service.rb

```diff
@@ -18,23 +19,25 @@ def start
         files = files_provider.pending_files
         in_progress = files_provider.processing_files
         stats[:pending] = files.length
-        stats[:progress] = in_progress.length
+        stats[:zombies] = in_progress.length
```

Comment:
do you refer to `:zombies` for those files pending? Or are for files that are stuck?